src/ImageManipulation.php

```diff
@@ -1104,6 +1101,7 @@ private function copyImageBackend(): AssetContainer
      */
     public function setAttribute($name, $value)
```

Comment:
The `ImageManipulation::setAttirbute()` has some custom logic not present in the `AttributesHTML` equivalent.